lib/command.js

```diff
@@ -186,6 +186,11 @@ class Command {
                 await preChecks(ui, system);
             }
 
+            ui.log('');
+            ui.log('Love open source? We’re hiring Node.js Engineers to work on Ghost full-time.', 'magentaBright');
```

Comment:
I don't _think_ it should - but have you checked to make sure this doesn't get output multiple times for commands that call other commands? (i.e. the `install` command runs `doctor` and `setup` under the hood)

Reply:
yeah that's what I figured, just wanted to be sure 🙂